pkg/serviceState/serviceState.go

```diff
@@ -0,0 +1,95 @@
+package servicestate
+
+import (
+	"os"
+
+	"github.com/datreeio/admission-webhook-datree/pkg/config"
+	"github.com/datreeio/admission-webhook-datree/pkg/enums"
+	"github.com/lithammer/shortuuid"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	WEBHOOK string = "Webhook"
+)
+
+type ServiceState struct {
+	clientId       string
+	token          string
+	clusterUuid    types.UID
+	clusterName    string
+	k8sVersion     string
+	policyName     string
+	isEnforceMode  bool
+	serviceVersion string
+	serviceType    string
+}
+
+func New() *ServiceState {
+
+	return &ServiceState{
+		clientId:       shortuuid.New(),
+		serviceType:    WEBHOOK,
+		token:          os.Getenv(enums.Token),
+		clusterName:    os.Getenv(enums.ClusterName),
+		policyName:     os.Getenv(enums.Policy),
+		isEnforceMode:  os.Getenv(enums.Enforce) == "true",
+		serviceVersion: config.WebhookVersion,
```

Comment:
no need for this to be generic, being specific makes it more readable
```suggestion
		webhookVersion: config.WebhookVersion,
```